sites/docs/docs/plugins/using-plugins/index.md

```diff
@@ -1,25 +1,32 @@
+---
+sidebar_position: 1
+hide_table_of_contents: false
+title: 'Using Plugins'
+---
+
 # Plugins
 
 Evidence uses a plugin system to load components (charts, buttons, and other visual parts of your project), and they are configured using the `evidence.plugins.yaml` file in your project.
 
 By default, new Evidence projects are configured with all the components needed to create most reports, but if you want to add others, this guide will cover methods for doing so.
 
 :::info
-If you want to create new plugins, see [Creating a Plugin](./creating-a-plugin).
+If you want to create new plugins, see [Creating a Plugin](/plugins/creating-a-plugin).
 :::
 
 ## Installing Plugins
 
-To install a plugin, use the `npm install` command. Make sure you include the `--save` flag or the plugin
-will not be saved with your project.
+To install a plugin, use the `npm install` command. Make sure you include the `--save` flag or the plugin will not be saved with your project.
 
 For example:
 
 ```bash
 npm install --save @acme/charting 
 ```
 
-Once the plugin is installed, you can add it to your plugin file and begin using it in your project
+Once the plugin is installed, you can add it to your plugin file and begin using it in your project.
+
+> **Note:** Adding new packages to your plugin file will not immediately take effect in your project. You will need to restart the Evidence server for the changes to take effect. This can be done by pressing the 'r' key in the terminal window where you have Evidence running.
```

Comment:
:::info
Changes or additions to your plugins file will not take effect in your project without restarting the Evidence server. This can also be done by pressing the 'r' key in the terminal window where you have Evidence running.
:::